etc/ci/check_intermittents.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+REPEAT_COUNT=100
+
+for test_type in wpt css; do
+    while read test_name; do
+        echo "  - Checking $test_name"
```

Comment:
It's inside a script called check_intermittents, and every CI system in the world shows which command was run before its actual stdout. The only reason I included `  - Checking` was to help distinguish it from the output of wptrunner itself.